src/magic.cpp

```diff
@@ -58,13 +58,13 @@ template<>
 std::string enum_to_string<spell_target>( spell_target data )
 {
     switch( data ) {
-        case spell_target::ally: return "ally";
-        case spell_target::hostile: return "hostile";
-        case spell_target::self: return "self";
-        case spell_target::ground: return "ground";
-        case spell_target::none: return "none";
-        case spell_target::item: return "item";
-        case spell_target::field: return "field";
+        case spell_target::ally: return translate_marker_context( "Valid spell target", "ally" );
+        case spell_target::hostile: return translate_marker_context( "Valid spell target", "hostile" );
+        case spell_target::self: return translate_marker_context( "Valid spell target", "self" );
+        case spell_target::ground: return translate_marker_context( "Valid spell target", "ground" );
+        case spell_target::none: return translate_marker_context( "Valid spell target", "none" );
+        case spell_target::item: return translate_marker_context( "Valid spell target", "item" );
+        case spell_target::field: return translate_marker_context( "Valid spell target", "field" );
```

Comment:
It might be better to create a separate function to return the translated string of these enums. The enum text may differ from the enum string id e.g. when an enum has an underscore in its name.

Reply:
Done.